tests/unit_tests/test_ustar_cp/test_cpdFindChangePoint20100901.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import matlab.engine
+import numpy as np
+
+@pytest.fixture(scope="module")
+def matlab_engine():
+    # Initialize MATLAB engine
+    eng = matlab.engine.start_matlab()
+    eng.addpath('/home/tcai/Projects/ONEFlux/oneflux_steps/ustar_cp')
```

Comment:
Same thing as before for the hardcoded path and matlab_engine

Reply:
Resolved